libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
Because this will almost certainly be called from another cookbook and I would prefer to make it obvious what you are resetting. For example:
```
::Chef::Recipe.send(:include, RabbitMQCookbook::Helpers)
# some code
# some more code
# finally call function
reset_this_node
```

This feels too ambiguous to me...